float64 ArmPosDes_1
float64 ArmPosDes_2
float64 ArmPosDes_3
float64 ArmPosDes_4
float64 ArmPosDes_5

